src/test/java/uk/gov/hmcts/reform/finrem/caseorchestration/controllers/NotificationsControllerTest.java

```diff
@@ -820,6 +820,46 @@ public void whenShouldNotSendContestedGeneralOrderEmailToRespondent_ThenTheEmail
         verify(notificationService, never()).sendContestedGeneralOrderEmailRespondent(any());
     }
 
+    @Test
+    public void whenToggleEnabledAndShouldSendEmailToRespSolicitor_thenSendsEmail() {
+        when(featureToggleService.isRespondentSolicitorEmailNotificationEnabled()).thenReturn(true);
+        when(notificationService.shouldEmailRespondentSolicitor(any())).thenReturn(true);
+
+        notificationsController.sendAssignToJudgeConfirmationEmail(AUTH_TOKEN, buildCallbackRequest());
+
+        verify(notificationService).sendAssignToJudgeConfirmationEmailToRespondentSolicitor(any());
```

Comment:
can you verify the caseDetails from buildCallbackRequest() is passed in this test

Reply:
done! 👍